fs.go

```diff
@@ -136,9 +136,12 @@ type File interface {
 	io.Seeker
 	io.Closer
 	// Lock locks the file like e.g. flock. It protects against access from
-	// other processes.
+	// other processes. The lock will be released by a call to Unlock or closing
+	// the file.
+	// Recursive locking (trying to lock the same file multiple times) or having
+	// multiple handles to a file may result in implementation specific behaviour.
```

Comment:
Our implementation expires the lock after a certain time, so this is a little bit different. But I'm not sure how to describe that in context of file locking/unlocking.